drydock/templates/drydock/k8s/newrelic.ini

```diff
@@ -0,0 +1,27 @@
+[newrelic]
+license_key = {{DRYDOCK_NEWRELIC_LICENSE_KEY }}
+app_name = {{ K8S_NAMESPACE }}
```

Comment:
I think most of these values are the default ones found here: https://docs.newrelic.com/docs/apm/agents/python-agent/configuration/python-agent-configuration/.

I think we can drop most of them, leave only `license_key` and `app_name`, and add a patch `newrelic-conf` or something like that.